CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -111,18 +118,29 @@ void Kartaclysm::StateRacing::Unsuspend(const int p_iPrevState)
 //------------------------------------------------------------------------------
 void Kartaclysm::StateRacing::Update(const float p_fDelta)
 {
+	// good shit
 	// Do not update when suspended
 	if (!m_bSuspended)
 	{
 		m_pGameObjectManager->Update(p_fDelta);
 	}
 
+	// bad shit
```

Comment:
"bad shit" doesn't help fix the problem.  Include why it is bad, and what should be changed to make it good shit.